enterprise/api/v1/views.py

```diff
@@ -171,7 +174,7 @@ def courses(self, request, pk=None):  # pylint: disable=invalid-name,unused-argu
         serializer.update_enterprise_courses(enterprise_customer, catalog_id=enterprise_customer.catalog)
         return get_paginated_response(serializer.data, request)
 
-    @detail_route(methods=['post'])
+    @detail_route(methods=['post'],  permission_classes=[IsAdminUserOrInGroup])
```

Comment:
do we know if using this overrides the permission classes for the view set or is this checked additionally?

Reply:
I was thinking that, DRF first checks for global permissions (those from the settings file), then for view permissions (declared in permission_classes -- if these exist, they will override global permissions) and only after that for method permissions (declared with the decorator @permission_classes) **but** It does not work like that. So here, It will override the permission_classes set on it before. 
So, We can define something like that for our POST call to `\course_enrollments\` endpoint. 

 ```
@detail_route(methods=['post'],  permission_classes=[
        permissions.IsAuthenticated,
        permissions.DjangoModelPermissions,
        IsEnterpriseEnrollmentAPIAccessed,
    ])
 def course_enrollments(self, request, pk):  # pylint: disable=invalid-name,unused-argument
    .....



```

Reply:
I think we should add permissions.IsAuthenticated, but leave out permissions.DjangoModelPermissions. Thanks for confirming.